cmd/vela-kubernetes/main.go

```diff
@@ -39,109 +39,109 @@ func main() {
-			Name:   "config.namespace",
-			Usage:  "kubectl namespace for interacting with Kubernetes",
+		&cli.StringFlag{
+			EnvVars: []string{"PARAMETER_NAMESPACE", "CONFIG_NAMESPACE", "KUBE_NAMESPACE"},
+			Name:    "config.namespace",
+			Usage:   "kubectl namespace for interacting with Kubernetes",
 		},
-		cli.StringFlag{
-			EnvVar: "PARAMETER_CONFIG_PATH,CONFIG_PATH,KUBE_CONFIG_PATH",
-			Name:   "config.path",
-			Usage:  "path to kubectl configuration file",
+		&cli.StringFlag{
+			EnvVars: []string{"PARAMETER_CONFIG_PATH", "CONFIG_PATH", "KUBE_CONFIG_PATH"},
+			Name:    "config.path",
+			Usage:   "path to kubectl configuration file",
 		},
 
 		// Patch Flags
 
-		cli.StringFlag{
-			EnvVar: "PARAMETER_CONTAINERS,PATCH_CONTAINERS",
-			Name:   "patch.containers",
-			Usage:  "containers from files to patch",
+		&cli.StringFlag{
+			EnvVars: []string{"PARAMETER_CONTAINERS", "PATCH_CONTAINERS"},
+			Name:    "patch.containers",
+			Usage:   "containers from files to patch",
 		},
-		cli.BoolFlag{
-			EnvVar: "PARAMETER_DRY_RUN,PATCH_DRY_RUN",
-			Name:   "patch.dry_run",
-			Usage:  "enables pretending to patch the containers",
+		&cli.BoolFlag{
+			EnvVars: []string{"PARAMETER_DRY_RUN", "PATCH_DRY_RUN"},
+			Name:    "patch.dry_run",
+			Usage:   "enables pretending to patch the containers",
 		},
-		cli.StringSliceFlag{
-			EnvVar: "PARAMETER_FILES,PATCH_FILES",
-			Name:   "patch.files",
-			Usage:  "kubernetes files or directories to patch",
+		&cli.StringSliceFlag{
+			EnvVars: []string{"PARAMETER_FILES", "PATCH_FILES"},
+			Name:    "patch.files",
+			Usage:   "kubernetes files or directories to patch",
 		},
-		cli.StringFlag{
-			EnvVar: "PARAMETER_OUTPUT,PATCH_OUTPUT",
-			Name:   "patch.output",
-			Usage:  "set output for patch - options: (json|yaml|wide)",
+		&cli.StringFlag{
+			EnvVars: []string{"PARAMETER_OUTPUT", "PATCH_OUTPUT"},
+			Name:    "patch.output",
+			Usage:   "set output for patch - options: (json|yaml|wide)",
 		},
 
 		// Status Flags
 
-		cli.StringSliceFlag{
-			EnvVar: "PARAMETER_STATUSES,STATUS_RESOURCES",
-			Name:   "status.resources",
-			Usage:  "kubernetes resources to watch status on",
+		&cli.StringSliceFlag{
+			EnvVars: []string{"PARAMETER_STATUSES", "STATUS_RESOURCES"},
+			Name:    "status.resources",
+			Usage:   "kubernetes resources to watch status on",
 		},
-		cli.DurationFlag{
-			EnvVar: "PARAMETER_TIMEOUT,STATUS_TIMEOUT",
-			Name:   "status.timeout",
-			Usage:  "maximum duration to watch status on kubernetes resources",
-			Value:  5 * time.Minute,
+		&cli.DurationFlag{
+			EnvVars: []string{"PARAMETER_TIMEOUT", "STATUS_TIMEOUT"},
+			Name:    "status.timeout",
+			Usage:   "maximum duration to watch status on kubernetes resources",
+			Value:   5 * time.Minute,
 		},
-		cli.BoolTFlag{
-			EnvVar: "PARAMETER_WATCH,STATUS_WATCH",
-			Name:   "status.watch",
-			Usage:  "enables watching the status until the rollout completes",
+		&cli.BoolFlag{
```

Comment:
Previously, this used to be a `BoolTFlag` from `urfave/cli/v1`.

The purpose of that flag type was to ensure we were using a bool that defaulted to `true`.

Would it be possible for you to add a `Value: true,` to this flag declaration? 

This should help maintain consistency for the `status.watch` flag defaulting to `true`?

Reply:
Absolutely, I had been doing some regex rules to make editing things easier and this slipped past, apologies.